### DISCUSSION TITLE
Is there a way to style labels and title for a plot?

### DISCUSSION THREAD

**Discussion by kdog3682** [observablehq/plot#1625] [Q&A]
Hello,

I would like to do the following things, but I am having some trouble:

1. Style the plot with css - ie give it a background-image, dotted border, et cetera
2. Give the plot a title (and be able to style the title with css)
3. Give the plot an x-axis label and a y-axis label and style them with css

Regarding 3, using axisX and axisY kind of works. But what seems to happen, is that the tick marks as well as the label get styled. I would like to be able to have different styles for both.

Thank you,
Kevin

**Comment by Fil** [comment]
In the [gallery](https://observablehq.com/@observablehq/plot-gallery) there is an [example](https://observablehq.com/@observablehq/plot-background-image) of (1) background image added with css.

For (2) you can create the [caption](https://observablehq.com/plot/features/plots#other-options) as an DOM node of any sort. It will appear below the chart. For a title you might want to set it at the top, in that case wrap your chart in a HTML node, for example with [htl](https://observablehq.com/@observablehq/htl) (or upvote https://github.com/observablehq/plot/issues/92).

For (3) axes you can create multiple [axes](https://observablehq.com/plot/marks/axis), say one that only has ticks, the other that only has the label, and style them differently.
